tasks/upgrade_component.yml

```diff
@@ -15,6 +15,8 @@
     remote_src: true
     dest: "/tmp/upgrade/{{ service_name }}/{{ item | basename }}-{{timestamp}}"
     mode: 0640
+    owner: "{{ owner }}"
+    group: "{{ group }}"
```

Comment:
Nitpick, but it seems like it is common practice to list the owner and group properties before the mode.

Reply:
ok good suggestion!